src/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ endif (PLUGIN_DENSE_PARSER)
 # Object library is necessary for jvm-package, which creates its own shared
 # library.
 if (USE_CUDA)
-  file(GLOB_RECURSE CUDA_SOURCES *.cu)
+  file(GLOB_RECURSE CUDA_SOURCES *.cu *.cuh)
```

Comment:
It's not usual to include headers since CMake will track those automatically.  I'm not sure if it's gonna affect CMake.  Is it only for source grouping?

Reply:
There is some discussion about it here: https://stackoverflow.com/questions/36174499/why-add-header-files-into-add-library-add-executable-command-in-cmake?rq=1

Header files don't get included in the IDE project unless they are included here. I don't see any reason not to do it